src/ray/common/runtime_env_manager.cc

```diff
@@ -35,6 +35,7 @@ const std::vector<std::string> &RuntimeEnvManager::GetReferences(
 }
 
 void RuntimeEnvManager::RemoveURIReference(const std::string &hex_id) {
+  RAY_LOG(DEBUG) << "Subtracting 1 from URI Reference for id " << hex_id;
```

Comment:
Can we remove this log? This will make log really spammy when debug log is on.

Reply:
This log is useful for debug now. And The log count is equal to starting worker number probably. Maybe it is acceptable for debug level logs？

Reply:
Hmm I see. If it matches to the number of worker processes, that'd be fine